src/main/webapp/app/shared/http/file-uploader.service.ts

```diff
@@ -33,7 +33,8 @@ export class FileUploaderService {
 
         const formData = new FormData();
         formData.append('file', file, fileName);
-        const url = options.keepFileName ? '/api/fileUpload?keepFileName=true' : '/api/fileUpload';
+        const keepFileName: boolean = !!options && options.keepFileName;
```

Comment:
Is the default for this value `true` or `false`?
Previous approach (`const url = options.keepFileName ? '/api/fileUpload?keepFileName=true' : '/api/fileUpload';`) did not work as the Http endpoint always requires the param `keepFileName` to be set.